package.json

```diff
@@ -45,5 +45,8 @@
     "query-string": "^6.1.0",
     "simple-oauth2": "^2.5.1"
   },
+  "resolutions": {
+    "graceful-fs": "^4.2.4"
```

Comment:
What's this for?

Reply:
This fixes a Reference error primordials is not defined which occurs when a dependency under gulp does that monkeypatch node.js.

This overrides and performs that monkeypatch 😄